.github/workflows/build-system.yml

```diff
@@ -0,0 +1,99 @@
+name: Continuous Integration v2
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  CI:
+    name: ${{ matrix.name }} - ${{ matrix.configuration }}
+    runs-on: ${{ matrix.os }}
+    container: ${{ matrix.container }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        configuration: [RelWithDebInfo, MinSizeRel]
```

Comment:
Switched to `MinSizeRel` and `RelWithDebInfo` to replace `Release` and `Debug`

Doesn't seem to cause issues running, however it appears it will cause longer build times on CI since the Conan packages don't have bins for these configs readily available.

Reply:
That is rather unfortunate. Remind me to look into making Dockerfiles for each configuration so we can just pre-build these.